packages/usa-select/src/styles/_usa-select.scss

```diff
@@ -30,6 +30,17 @@
     background-image: none;
   }
 
+  &:disabled,
+  &[aria-disabled="true"] {
+    @include u-disabled;
+    opacity: 1; // Undo chromium browser style
+
+    option {
+      color: color(ink); // Undo browser style
+      opacity: 0.7;
```

Comment:
Again, just a note on consistency of value. We should try to stick with either `0.6` or `0.7`.

Reply:
I originally put this as 0.7 because it matched chrome disabled styles for all browsers. It appears that `option` is not targetting the select text like I thought it was. This was displaying the text as `ink` with no opacity shift.

For the time being, I've changed this to `.8` as this seems to be the magic opacity dark grey text on the disabled background color and matches my suggestion for file input up above.

It looks like: 
- `opacity: 0.6` is the golden number for disabled elements with a white background
- `opacity: 0.8` for elements with text on a disabled background

### Screenshots

Before:
![image](https://user-images.githubusercontent.com/25211650/211603514-8b854054-89b6-4013-a925-a73d77d6c79d.png)


After:
![image](https://user-images.githubusercontent.com/25211650/211602837-f7a6bc44-abab-4632-b1a3-c4efa487a067.png)

Passes on:
- [x] : Chrome
- [x] : Firefox
- [x] : Safari

### Disadvantage
This does adjust the opacity of the background as well, so potentially causing inconsistency between disabled colors on inputs.